frontend/src/store/root-reducer.ts

```diff
@@ -1,7 +1,9 @@
 import { reducer as auth } from './auth/reducer';
+import { reducer as users } from './users/reducer';
```

Comment:
```suggestion
import { reducer as uam } from './uam/reducer';
```
each reducer should be per page